lstchain/calib/camera/pulse_time_correction.py

```diff
@@ -152,7 +152,9 @@ def get_first_capacitor(self, event, nr):
 
 @njit()
 def get_corr_time_jit(first_cap, fan, fbn, n_harmonics, n_cap):
-    time = fan[0] / 2.
+
+    #time = fan[0] / 2. #commented because time flat-fielding is performed in charge calibrator
+    time = 0
```

Comment:
Why here time is set to 0?

Reply:
Hi @pawel21,
it is written in the commented line above. As far as i understood this is the flat-field coefficient (can you confirm this?) and we decided to perform it later on at charge calibration level.

Reply:
This is flat-field coefficient in time domain.

Reply:
so is it correct or not?

Reply:
It is correct for me, `fan[0] / 2`is constant factor, so this will only change value of pulse time, but shouldn't affect time correction.